apps/crn-server/test/fixtures/research-output.fixtures.ts

```diff
@@ -150,7 +154,9 @@ export const getSquidexGraphqlResearchOutput = (): NonNullable<
   },
 });
 
-export const getResearchOutputDataObject = (): ResearchOutputDataObject => ({
+export const getResearchOutputDataObject = ({
+  published = true,
```

Comment:
we don't generally use the pattern of overrides as parameters for fixtures in BE - we decided it's easier to mutate the fixture in the test. I would just leave the function as it was, have the `published` property set to `true` but in your test just do 
```
const researchOutputDataObject = getResearchOutputDataObject();
researchOutputDataObject.published = false
```
I think in your test you actually need a list so you would have to reference an element in the list but that should be fine too

Reply:
ok will revert this change and mutate in the test